floodwatch/src/js/components/Main.js

```diff
@@ -3,7 +3,7 @@ import React, {Component} from 'react';
 import {Link} from 'react-router';
 import {Grid, Nav, Navbar, NavItem, Row, Col} from 'react-bootstrap';
 
-import '../../css/app.css';
+import '../../css/App.css';
```

Comment:
This should be lowercase... the new css build rules output to lowercase, so maybe you need to restart that process